modules/aleph/includes/aleph.reservation.inc

```diff
@@ -37,7 +111,28 @@ function aleph_reservation_reservation_deletion_enabled() {
   return FALSE;
 }
 
-function aleph_reservation_branch_name() {
+/**
+ * Return a branch name for a given branch id.
+ *
+ * @param string $branch_id
+ *   The identifier for a given pick-up branch (e.g. BBAAA).
+ *
+ * @return string
+ *   The human name for the id or NULL if one do not exist.
+ */
+function aleph_reservation_branch_name($branch_id) {
+  if (strlen($branch_id) > 5) {
```

Comment:
Why this? Is the branch_id in fact the human-readable name in some cases? Then it needs to be documented.